src/server/pfs/fuse/BUILD.bazel

```diff
@@ -59,17 +49,14 @@ go_test(
         "no-sandbox",
     ],  # fusermount cannot run in bazel test sandbox or docker
     deps = [
-        "//src/auth",
         "//src/internal/client",
         "//src/internal/dockertestenv",
         "//src/internal/errors",
         "//src/internal/pctx",
         "//src/internal/require",
         "//src/internal/testpachd/realenv",
-        "//src/internal/testutil",
         "//src/internal/testutil/random",
         "//src/pfs",
-        "//src/pps",
```

Comment:
how come pps is getting removed here?

Reply:
After removing mount server code, that dependency is no longer used in the fuse package.